extensions/sql-bindings/src/common/constants.ts

```diff
@@ -77,7 +77,12 @@ export const connectionProgressTitle = localize('connectionProgressTitle', "Test
 export const enterObjectName = localize('enterObjectName', 'Enter SQL table or view to query');
 export const enterObjectNameToUpsert = localize('enterObjectNameToUpsert', 'Enter SQL table to upsert into');
 export const selectTable = localize('selectTable', 'Select table to use');
+export const selectView = localize('selectView', 'Select view to use');
 export const tableListProgressTitle = localize('tableListProgressTitle', "Fetching tables for selected database...");
+export const viewListProgressTitle = localize('viewListProgressTitle', "Fetching views for selected database...");
+export const sqlTable = localize('sqlTable', 'SQL table');
```

Comment:
I think just `Table` and `View` are fine here - they already know that they're connecting to a MSSQL database